pkg/lightning/backend/local/duplicate.go

```diff
@@ -997,7 +998,7 @@ func (m *dupeDetector) processRemoteDupTask(
 			}
 			return nil
 		}
-		if log.IsContextCanceledError(err) {
+		if stderrors.Is(err, context.Canceled) {
```

Comment:
will err be nested? maybe check `ctx.Err() != nil`

Reply:
I see https://pkg.go.dev/errors#Is can handle nested error by checking `Unwrap` interface. However I think check the `ctx` argument is more direct. I'll check it soon.